presto-main/src/main/java/io/prestosql/sql/rewrite/ShowQueriesRewrite.java

```diff
@@ -231,10 +227,6 @@ protected Node visitShowGrants(ShowGrants showGrants, Void context)
 
                 catalogName = qualifiedTableName.getCatalogName();
 
-                accessControl.checkCanShowTablesMetadata(
```

Comment:
Can you file an issue for this one?  I believe we should have some kind of access control on `SHOW GRANTS FOR TABLE`.  My guess would be you can always see your own grants, or if you are the owner, you can see all grants... or maybe the "with grant option".  We'll need to look it up in the spec.

Reply:
Currently, it is only implemented for `hive.security=sql-standard`. If you have `SET ROLE admin` then you display all table privileges, otherwise you display only session user grants.

Reply:
Maybe we should have something like `filterTablePrivileges` or `filterGrants`. The same thing goes to listing `roles` I think. In both cases, it is currently handled in `sql-standard` access control directy.

Reply:
Having that said, I don't think there is anything actionable to be logged for today.